packages/server-wallet/src/db/migrations/20201014174040_close-channel-objectives.ts

```diff
@@ -0,0 +1,17 @@
+import * as Knex from 'knex';
+
+const tableName = 'close-channel-objectives';
+
+export async function up(knex: Knex): Promise<any> {
+  return knex.schema.createTable(tableName, function(table) {
```

Comment:
This table has identical schema to `open-channel-objectives`. What's the purpose of having two separate tables?

Reply:
It's not exactly the same, although it is very similar. In general, the different objective types have `data` properties of differing shape.

Reply:
That's on the client side though. The DB schema appears identical to me.

Reply:
Yeah you're right, but it was a typo 🥴 . The close channel objective shouldn't have a `fundingStrategy`.

Reply:
The funding strategy sounds like it belongs in the field to me.

Reply:
Hmm maybe. I was going off of the type recently added to `wallet-core` on the base branch 
https://github.com/statechannels/statechannels/blob/e0f75ef9236548f0fcc78db05bd58472c27292ea/packages/wallet-core/src/types.ts#L102-L106